pyproject.toml

```diff
@@ -0,0 +1,4 @@
+[build-system]
```

Comment:
With a `pyproject.toml` file we are using the latest python packaging guidelines. This "future proofs" against some changes that will be rolled into `pip` soon and ensures that the setup dependencies are installed at setup time.